src/rulesReport.ts

```diff
@@ -98,10 +98,48 @@ export const reportRules = (groupBy: GroupBy) => {
   processOffenses(reportShortVariableName)
   processOffenses(reportTooManyProps)
 
-  const health: { file: string, errors: number, warnings: number }[] = []
+  const health: Health[] = []
+
+  // Sort offenses grouped by key based on the `orderBy` parameter
+  const sortedKeys = Object.keys(offensesGrouped).sort((a, b) => {
+    const countA = offensesGrouped[a].length
+    const countB = offensesGrouped[b].length
+
+    if (orderBy === 'desc') {
+      return countB - countA
+    }
+
+    return countA - countB
+  })
 
   // Output the report grouped by file
-  Object.keys(offensesGrouped).forEach((key) => {
+  // Object.keys(offensesGrouped).forEach((key) => {
```

Comment:
Do we want to keep all this commented out code?

Reply:
My bad 👀 that was while testing the new approach. I'll fix all comments tomorrow 🫡

Reply:
Removed in latest commit